LanguageExt.CodeGen/LanguageExt.CodeGen.csproj

```diff
@@ -1,16 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Build">
-
-  <PropertyGroup>
-    <RoslynCodeGenVersion>0.6.1</RoslynCodeGenVersion>
-  </PropertyGroup>
-  
-  <!--<Sdk Name="CodeGeneration.Roslyn.Plugin.Sdk" Version="0.7.5-alpha" />-->
+  <Sdk Name="CodeGeneration.Roslyn.PluginMetapackage.Sdk" Version="0.7.63" />  
 
   <PropertyGroup>
     <PackageVersion>3.4.13</PackageVersion>
     <PackageId>LanguageExt.CodeGen</PackageId>
     <Title>LanguageExt.CodeGen</Title>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
I think this can target `netstandard1.0`, same as Attributes package - you'll gain wider compatibility/applicability.

Reply:
Changed!